package.json

```diff
@@ -12,15 +12,32 @@
     "url": "https://github.com/eventualbuddha/keysim.js"
   },
   "scripts": {
-    "build": "gobble build dist -f",
+    "prebuild": "mkdir -p dist",
+    "build:node": "npm run prebuild && babel lib/keysim.es > dist/keysim.node.js",
+    "build:browser": "npm run prebuild && browserify . --standalone keysim > dist/keysim.browser.js",
+    "build": "npm run build:node && npm run build:browser",
     "test": "npm run test:node && npm run test:browser",
-    "test:browser": "./node_modules/karma/bin/karma start karma.conf.js",
-    "test:node": "mocha -R spec --recursive test"
+    "test:browser": "karma start --single-run",
+    "test:node": "mocha --require babel-register -R spec --recursive test",
+    "prepublish": "in-publish && npm test && npm run build || not-in-publish"
+  },
+  "browser": "lib/keysim.es",
```

Comment:
This means that if a user uses browserify himself to bundle keysim as a dependency, his browserify will use the source  as an entry point. It should work with the browserify settings that we've specified, including the use of babel.

I see it as more correct than not specifying this field, which will leave browserify using the `main` field, which is the build for node.

Also, this is another thing that rollup can't yet do, I would guess.